packages/v3/src/esp-entity-table.ts

```diff
@@ -271,30 +283,75 @@ class ActionRenderer {
     </button>`;
   }
 
+  private _tempSelector(entity: entityConfig, target: string) {
+    if (!entity) return;
+    let targetTemp =
+      target === "high"
+        ? entity.target_temperature_high
+        : entity.target_temperature || entity.target_temperature_low;
```

Comment:
One small improvement might be caching this choice to something like `const lowTemp = entity.target_temperature || entity.target_temperature_low;` and using that reference instead. Probably same for most of the `Number` casts used multiple times.